src/cmd/extrinsics/runtime_api/mod.rs

```diff
@@ -15,6 +15,17 @@
 // along with cargo-contract.  If not, see <http://www.gnu.org/licenses/>.
 
 #[subxt::subxt(
-    runtime_metadata_path = "src/cmd/extrinsics/runtime_api/contracts_runtime.scale"
+    runtime_metadata_path = "src/cmd/extrinsics/runtime_api/contracts_runtime.scale",
+    derive_for_type(type = "sp_runtime::DispatchError", derive = "::serde::Deserialize"),
```

Comment:
Adding these derives to these generated types is necessary so that they can be used to deserialize the RPC `--dry-run` error responses.